.github/workflows/publish.yaml

```diff
@@ -0,0 +1,49 @@
+name: Publish ROCK
+on:
+  push:
+    branches:
+      - '*.*/stable'
+  workflow_dispatch:
+    branches:
+      - '*.*/stable'
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+  publish:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Docker
+        run: |
+          sudo snap install docker
+          sudo addgroup --system docker; sudo adduser $USER docker
+          newgrp docker
+          sudo snap disable docker; sudo snap enable docker
+      - name: Install skopeo
+        run: |
+          sudo snap install --devmode --channel edge skopeo
+      - name: Install yq
+        run: |
+          sudo snap install yq
+      - uses: actions/download-artifact@v3
+        with:
+          name: charmed-kafka
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Import and push to Docker Hub
+        run: |
+          version="$(cat rockcraft.yaml | yq e '.version')"
+          sudo skopeo \
+            --insecure-policy \
+            copy \
+            oci-archive:"charmed-kafka_${version}_amd64.rock" \
+            docker-daemon:"charmed/charmed-kafka:${version}"
+          docker push "charmed/charmed-kafka:${version}"
```

Comment:
The repo is https://hub.docker.com/r/charmed/kafka
Currently you are trying to push into https://hub.docker.com/r/charmed/charmed-kafka

AFAIK, @delgod idea is to have `charmed/kafka` repo name with image name `charmed-kafka`.

Reply:
it was my understanding that we would change this to "dataplatformoci". Isn't this the case?